devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -58,6 +58,7 @@ metadata {
 }
 
 def installed() {
+    updateDataValue("updatedOnOff", "true")
```

Comment:
We decided to change the key to `"onOff"` and the value to `"catchall"`. Also put this in `updated` instead of installed. Finally the PR should be against staging since that's where the DTH was merged.